src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,7 @@
+package duke;
+
+public class Ui {
```

Comment:
Should your Ui do more than just print out Strings?

Reply:
yeah i'll throw in some input or sth